### ISSUE TITLE
A11y_VSCode_ChangingoftheTheme_ScreenReader: Screen reader does not announce the drop down list items of 'select color theme' combo box after selecting them.

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode#76868]
**Environment Details:**
VSCode Version : 1.36.1 (user setup)
OS Version : MacOS Mojave 10.14.5

**Additional Details:**
MAS Violated : MAS 1.3.1
Screen Reader Versions : Voice over

**Repro Steps:**
1.Open Visual studio code editor.
2.Open the command Pallet by pressing "F1" key and type "Theme"
3.Using Down arrow key select "Preference colour Theme" and press enter key to select.
4.Using "Up/Down" arrow key select different available theme.
5.Start screen reader.

**Actual:**
Screen reader does not announce the drop down list item names of select 'colour theme combo box 'after selecting them

This issue is observed in Mac + voice over

**Expected:**
Screen reader should announce the drop down item name after selecting it.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not understand currently which theme is selected.

**MAS Reference:**
**MAS 1.3.1 -** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference:
[A11y_VSCode_ChangingoftheTheme_ScreenReader.pptx](https://github.com/microsoft/vscode/files/3368017/A11y_VSCode_ChangingoftheTheme_ScreenReader.pptx)

**Comment by isidorn** [comment]
Disagree. We read before choosing the theme, not after.
The user is aware of what was selected before he choose it since we announce each choice.

If you disagree with me please provide an example of some MS software which reads both before and after user makes a choice. Thanks
Closing as invalid
